CMSgov/design-system
# [WNMGDS-963] updating vertical nav for improved accessibility

## Summary
Vertical Nav accessibility update

### Changed
Vertical Nav component to have improved accessibility by wrapping it in a `<nav>` element and adding an optional `aria-label` prop.

## How to test
Access the demo site at http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-963/vertical-nav-accessibility-update

Navigate to the Vertical Nav component

Examine the React iframe and confirm that the <nav> element now wraps the component & an aria-label is present (only because it is configured that way)

The docs for the VerticalNav should also reflect the prop addition. Confirm the information provided is acceptable for consumers
